### PR TITLE
sd-radv: allow to configure reachable time

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: man/systemd.network.xml [hunk 1]
```diff
@@ -4000,6 +4000,18 @@ ServerAddress=192.168.0.1/24</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ReachableTimeSec=</varname></term>
+
+        <listitem>
+          <para>Configures the reachable time, used by by the Neighbor Unreachability Detection algorithm
+          that clients assume a neighbor is reachable after having received a reachability confirmation.
+          Takes a timespan in the range 0…4294967295 msec. Defaults to 0.</para>
```
Comment (keszybz) [inline man/systemd.network.xml]: "Configures the time, used in the Neighbor Unreachability Detection algorithm, for which clients can assume a neighbor is reachable after having received a reachability confirmation. Time a time span in the range  0…4294967295 ms. Defaults to 0."

(Please use "ms". "ms" is the SI unit.)

What does it mean when 0 is used? The code below ignores that value, so maybe this should say "When set to 0, this option is not sent to clients. Defaults to 0.".

**Review by keszybz — CHANGES_REQUESTED** [review]

File: man/systemd.network.xml (hunk 1 above)
Comment (yuwata) [reply man/systemd.network.xml]: Thanks. Applied the suggested sentence.

The value is used in RA header. So, 0 is also a valid value, and will be sent to clients as is. When a client receives RA with 0 reachable time, then the value will be handled as unspecified and will be ignored.

File: man/systemd.network.xml [hunk 2]
```diff
@@ -4000,6 +4000,19 @@ ServerAddress=192.168.0.1/24</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ReachableTimeSec=</varname></term>
+
+        <listitem>
+          <para>Configures the time, used in the Neighbor Unreachability Detection algorithm, for which
+          clients can assume a neighbor is reachable after having received a reachability confirmation. Takes
+          a time span in the range 0…4294967295 ms. When 0, clients will handle it as the time is
```
Comment (keszybz) [inline man/systemd.network.xml]: "When 0, clients will handle it as if the value wasn't specified."

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by yuwata** [comment]
@keszybz Thank you for the review. The suggestion applied. Upgrading the green label.

**Comment by bluca** [comment]
Looks unrelated but new @mrc0mmand 

`11:49:05 [   10.464173] systemd-udevd[513]: Failed to initialize libkmod context: Operation not supported`

**Pull request merged**